service/app.py

```diff
@@ -0,0 +1,103 @@
+import os
+import subprocess
+import uuid
+
+from flask import Flask
+from flask_restful import Resource, Api, reqparse
+
+import werkzeug
+
+# Set up the flask app and API
+app = Flask(__name__)
+api = Api(app)
+
+# Declare two global variables that will hold state.
+WORKFLOW_TESTS = {}
+ENVIRONMENTS = {}
+
+# Functions for running the test
+def submit_test(environment_image, workflow_source, workflow_inputs, workflow_dependencies):
+
+    docker_cmd = ["docker", "run", "-v /io:/io", "-v /var/run/docker.sock:/var/run/docker.sock", environment_image]
+    docker_cmd.extend(["--workflowSource", workflow_source])
```

Comment:
is there a reason you don't multi-line this in one literal instead of extending?